src/include/daos/debug.h

```diff
@@ -1,5 +1,6 @@
 /**
- * (C) Copyright 2015-2023 Intel Corporation.
+ * (C) Copyright 2015-2024 Intel Corporation.
```

Comment:
why are you editing the Intel copyright?

Reply:
that one makes no sense (unless he was an Intel employee when starting this PR)

Reply:
Yes, I was. This PR is almost half a year old.